### ISSUE TITLE
⚙️ Improve accessibility of main menu.

### ISSUE THREAD

**Issue by schalkneethling** [mdn/mdn-fiori#238]
The main menu is a *lot* more accessible than it used to be, but there is still one big aspect that can be improved. 

While you can currently tab through all the elements of the main menu, you cannot use the arrow keys to navigate through the submenu items once a submenu is activated. Also, while you can tab through these items, the focus is not constrained to the submenu items.

- [ ] Pressing the space bar does not currently open the submenu
- [ ] When a submenu is opened, move focus to the first item
- [ ] Navigate submenu items using the arrow keys

See https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html for more details on implementation
